README.md

```diff
@@ -167,8 +171,48 @@ jobs:
           password: ${{ secrets.GCR_JSON_KEY }}
 ```
 
+#### Workload identity federation based authentication
+
+Configure the workload identity federation for github actions in gcloud (for steps, [refer here](https://github.com/google-github-actions/auth#setting-up-workload-identity-federation)). In the steps, your service account should the ability to push to GCR. Then use google-github-actions/auth action for authentication using workload identity like below:
+
+```yaml
+name: ci
+
+on:
+  push:
+    branches: master
+
+jobs:
+  login:
+    runs-on: ubuntu-latest
+    steps:
+      - id: 'auth'
+      name: 'Authenticate to Google Cloud'
```

Comment:
I think this indentation is off by two spaces for this entire section

Reply:
Updated